<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>110</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>CameraConfig</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="x">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>摄像头横向调整</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_x">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="y">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>摄像头纵向调整</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_y">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="z">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>摄像头高低调整</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_z">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="tar_x">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>画面横向调整</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_tarX">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="tar_y">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>画面纵向调整</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_tarY">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="cam_f">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>焦距F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_camF">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Record</string>
    </property>
    <addaction name="StartRecord"/>
    <addaction name="StopRecord"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="OpenFile"/>
    <addaction name="OpenDataFile"/>
   </widget>
   <widget class="QMenu" name="menuCameraCfg">
    <property name="title">
     <string>CameraCfg</string>
    </property>
    <addaction name="OpenCamera"/>
    <addaction name="CloseCamera"/>
   </widget>
   <widget class="QMenu" name="menuDriver">
    <property name="title">
     <string>Driver</string>
    </property>
    <addaction name="ESRCan"/>
    <addaction name="ZLGCan"/>
   </widget>
   <addaction name="menuCameraCfg"/>
   <addaction name="menuDriver"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuRecord"/>
  </widget>
  <action name="OpenFile">
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="StartRecord">
   <property name="text">
    <string>StartRecord</string>
   </property>
  </action>
  <action name="StopRecord">
   <property name="text">
    <string>StopRecord</string>
   </property>
  </action>
  <action name="OpenCamera">
   <property name="text">
    <string>OpenCamera</string>
   </property>
  </action>
  <action name="CloseCamera">
   <property name="text">
    <string>CloseCamera</string>
   </property>
  </action>
  <action name="OpenDataFile">
   <property name="text">
    <string>OpenDataFile</string>
   </property>
  </action>
  <action name="ESRCan">
   <property name="text">
    <string>ESRCan</string>
   </property>
  </action>
  <action name="ZLGCan">
   <property name="text">
    <string>ZLGCan</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
